plugins/ops/log-streamer/test/integration/test_it_log_file_handler.py

```diff
@@ -10,10 +10,10 @@
 
 
 @pytest.mark.asyncio
-@pytest.mark.skipif(
-    platform.system().lower() != "linux",
-    reason="LogFileReader uses watchdog that works with no additional helpers only in Linux"
-)
+# @pytest.mark.skipif(
```

Comment:
Why this is commented out?

Reply:
Commented out because tests now run on macOS